ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision/Deadline.hs

```diff
@@ -0,0 +1,450 @@
+{-# LANGUAGE BangPatterns        #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE TypeOperators       #-}
+
+-- | This module contains the part of the block fetch decisions process that is
+-- specific to the deadline mode.
+module Ouroboros.Network.BlockFetch.Decision.Deadline
```

Comment:
Are there any changes worth reviewing in this module?

Reply:
No, everything has just been extracted from `Decision.hs` at the beginning and not touched since.